packages/@okta/vuepress-site/conductor.yml

```diff
@@ -430,7 +430,7 @@ redirects:
   - from: /use_cases/relationships/index.html
     to: /docs/reference/api/linked-objects/
   - from: /docs/api/getting_started/releases-at-okta/index.html
-    to: /docs/reference/releases-at-okta/
+    to: https://developer.okta.com/docs/api/openapi/okta-management/guides/release-lifecycle/
```

Comment:
This is great! Thanks for editing the redirect file.
For completeness, you might want to test these redirects on the staging branch. Whenever we update the conductor.yml file, it's always good to test it. Sometimes a trivial change can have some funky, unintended behaviour.